### PR TITLE
Update info to 2022 year

### PR DESCRIPTION
**Pull request by Drsheppard01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Awesome Flatpak
+# Awesome Flatpak  [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
```
Comment (akvadrako) [inline README.md:1]: This repo isn't on awesome.re though. If they add us, then we can have the link.

File: README.md [hunk 2]
```diff
@@ -29,15 +29,20 @@ Others:
...
 ## Tools
-
+### App Stores
+- [Souk](https://gitlab.gnome.org/haecker-felix/souk) - Souk is a flatpak-based App Store, written with GTK4 and Rust.
+- [bauh](https://github.com/vinifmor/bauh) - Graphical user interface for installing Linux applications. Supports AppImage, Arch packages (AUR, Pacman), Debian packages, Flatpak, Snap and native Web applications
+- [Zorin Store](https://zorin.com/os/) - Zorin OS Store comes pre-loaded with the app catalogs of Flathub, the Snap store, and the Ubuntu & Zorin OS APT repositories, AppImage packages and Windows apps (optionally using Windows App Support, powered by WINE)
+### Another Tools
 - [Flatseal](https://flathub.org/apps/details/com.github.tchx84.Flatseal) -  Permissions manager for Flatpak Apps
-- rpm2flatpak
-- [game-to-flatpak](https://github.com/hadess/flatpak-games) - 2018
```
Comment (akvadrako) [inline README.md:35]: Why remove rpm2flatpak and game-to-flatpak?

File: README.md [hunk 3]
```diff
@@ -29,15 +29,20 @@ Others:
 - [skype](https://github.com/snaggen/skype-app) -  2017
 
 ## Tools
-
+### App Stores
+- [Souk](https://gitlab.gnome.org/haecker-felix/souk) - Souk is a flatpak-based App Store, written with GTK4 and Rust.
+- [bauh](https://github.com/vinifmor/bauh) - Graphical user interface for installing Linux applications. Supports AppImage, Arch packages (AUR, Pacman), Debian packages, Flatpak, Snap and native Web applications
+- [Zorin Store](https://zorin.com/os/) - Zorin OS Store comes pre-loaded with the app catalogs of Flathub, the Snap store, and the Ubuntu & Zorin OS APT repositories, AppImage packages and Windows apps (optionally using Windows App Support, powered by WINE)
+### Another Tools
```
Comment (akvadrako) [inline README.md:36]: Another Tools → Other Tools

**Review by akvadrako — CHANGES_REQUESTED** [review]
Hi @Drsheppard01. Thanks for the first PR! I welcome the additions but:

1. I don't want to remove links, especially to still existing tools
2. We don't meet the awesome.re requirements, so unless they accept our repo anyway, I don't want to add a link to them.

File: README.md (hunk 2 above)
Comment (Drsheppard01) [reply README.md:35]: cause this looks like obandoned project

File: README.md (hunk 1 above)
Comment (Drsheppard01) [reply README.md:1]: okay, i add this list in [awesome.re](https://awesome.re/)

File: README.md (hunk 3 above)
Comment (Drsheppard01) [reply README.md:36]: Agreed, but unfortunately I don't know how to fix it now

**Comment by akvadrako** [comment]
Thanks, I've merged it with some of the changes I mentioned.

**Pull request merged**